dev-libs/libedit/libedit-20170329.3.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit eutils flag-o-matic toolchain-funcs versionator multilib-minimal
+
+MY_PV=$(get_major_version)-$(get_after_major_version)
+MY_P=${PN}-${MY_PV}
+
+DESCRIPTION="BSD replacement for libreadline"
+HOMEPAGE="http://www.thrysoee.dk/editline/"
+SRC_URI="${HOMEPAGE}${MY_P}.tar.gz"
```

Comment:
Please don't use HOMEPAGE here. By definition, HOMEPAGE can list multiple sites and reusing it as a single value is contrary to the principle of least surprise ;-).